hack/lib/build/version.sh

```diff
@@ -26,6 +26,19 @@ readonly -f os::build::version::get_vars
 function os::build::version::openshift_vars() {
 	local git=(git --work-tree "${OS_ROOT}")
 
+	if [[ -z ${OS_GIT_CATALOG_VERSION-} ]]; then
+		# search git merge commits for template text and extract version
+		# subject template: Merge version v0.0.14 of Service Catalog from https://github.com/openshift/service-catalog:v0.0.14+origin
+		local summary_text=$(${git[@]} log --merges --grep "Merge version v.* of Service Catalog from https://github.com/openshift/service-catalog" --pretty=%s -1)
+		if [[ $summary_text =~ Merge[[:space:]]version[[:space:]](v.*)[[:space:]]of[[:space:]]Service[[:space:]]Catalog ]]; then
```

Comment:
```sh
if [[ "${summary_text}"
      ^ ^            ^^
```

Reply:
Since you have the `grep` above in the `git` call you can probably make this less busy if you want

```sh
if [[ "${summary_text}" =~ .*[[:space:]](v.*)[[:space:]].* ]]; then
```